.github/CODEOWNERS

```diff
@@ -17,3 +17,4 @@
 /traffic_director/      @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /vertex_ai/             @terraform-google-modules/dee-data-ai @terraform-google-modules/terraform-samples-reviewers
 /vpc/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
+/bigquery/              @terraform-google-modules/terraform-samples-admins @terraform-google-modules/terraform-samples-reviewers
```

Comment:
Hi, I see that only terraform-samples teams are added as owners.
Q: Is there any better option? Any `dee-` teams which understand & regular works on BigQuery Samples?